src/Text/LLVM/AST.hs

```diff
@@ -1411,6 +1419,63 @@ data FCmpOp = Ffalse  | Foeq | Fogt | Foge | Folt | Fole | Fone
     deriving (Data, Eq, Enum, Generic, Ord, Show, Typeable)
 
 
+-- Debug Instructions ----------------------------------------------------------
+
+data DebugRecord' lab
+  = DebugRecordValue (DbgRecValue' lab)
+  | DebugRecordDeclare (DbgRecDeclare' lab)
+  | DebugRecordAssign (DbgRecAssign' lab)
+  | DebugRecordValueSimple (DbgRecValueSimple' lab)
+  | DebugRecordLabel (DbgRecLabel' lab)
+  deriving (Data, Eq, Functor, Generic, Ord, Show, Typeable)
+
+data DbgRecValue' lab = DbgRecValue
+  {
+    drvLocation :: ValMd' lab -- DILocation
+  , drvLocalVariable :: ValMd' lab -- DILocalVariable
+  , drvExpression :: ValMd' lab -- DIExpression
+  , drvValAsMetadata :: ValMd' lab -- Metadata*
```

Comment:
Perhaps these comments should be converted to Haddock comments? (Similarly for `DbgRecValueSimple'`, `DbgRecDeclare'`, `DbgRecAssign'`, and `DbgRecLabel'` below.)